object-cache.php

```diff
@@ -561,7 +561,7 @@ public function get( $_key, $group = 'default', $force = false ) {
 
 			if ( isset( $this->to_unserialize[ $redis_key ] ) ) {
 				unset( $this->to_unserialize[ $redis_key ] );
-				$value = unserialize( $value );
+				$value                         = maybe_unserialize( $value );
```

Comment:
This can actually spawn more errors. Because unserialize returns an empty value and a warning that something is wrong, which is arguably safer and more predictable than what maybe_unserialize returns - the value itself which is unexpected.

Moved to #19